Drop-In/src/test/java/com/braintreepayments/api/DropInRequestUnitTest.java

```diff
@@ -140,6 +140,7 @@ public void isParcelable() {
         billingAddress.setCountryCodeAlpha2("US");
         billingAddress.setPhoneNumber("3125557890");
         threeDSecureRequest.setBillingAddress(billingAddress);
+        threeDSecureRequest.setCardAddChallengeRequested(false);
```

Comment:
Is this needed? I thought this param was optional

Reply:
Yeah this was the bug that got fixed [#429](https://github.com/braintree/braintree_android/pull/429). It was throwing a null pointer exception.